package.json

```diff
@@ -30,6 +30,12 @@
 		"test": "grunt",
 		"grunt": "grunt",
 		"mocha": "mocha",
+		"jest:unit":
```

Comment:
Isn't there a sustainable way of adding scripts to `package.json`? If we continue this fashion nobody is gonna find the useful scripts.

Besides, the user doesn't receive any feedback/aid right now if they type the wrong suite. `npm run jest:wrong`. It's not a stopper but we have this before.

Reply:
Unfortunately not. I don't want to add another script to execute scripts while we can just simply run npm scripts.

> Besides, the user doesn't receive any feedback/aid right now if they type the wrong suite. `npm run jest:wrong`. It's not a stopper but we have this before.

The scripts are npm scripts. If you try to run an non-existing script npm complains.
![image](https://user-images.githubusercontent.com/26697004/53340422-48c6fc80-3909-11e9-9189-a65abad1076a.png)